code/modules/onestar/os_turret.dm

```diff
@@ -265,6 +273,11 @@
 	)
 	var/target_superior_mobs = FALSE
 
+/obj/item/electronics/circuitboard/os_turret/examine(user, distance)
+	. = ..()
+	if(target_superior_mobs)
+		to_chat(user, SPAN_NOTICE("When constructed, this turret will target roaches, spiders, and golems."))
```

Comment:
Wouldn't nearly identical text be added by `description_info`?

Reply:
It’s supposed to let people know when the circuitboard will build a friendly turret. The description info just says a reconstructed turret will do so.